extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/PanelStudyBrowserTracking.tsx

```diff
@@ -479,7 +479,11 @@ function _mapDisplaySets(
                   switch (action.id) {
                     case 'yes':
                       try {
-                        await dataSource.reject.series(ds.StudyInstanceUID, ds.SeriesInstanceUID);
+                        await dataSource.reject.series(
+                          ds.StudyInstanceUID,
+                          ds.SeriesInstanceUID,
+                          ds.instances[0].clientName
```

Comment:
ds.instance.RetrieveAETitle
You should have a single instance that defines the series rather than expecting instance 0 to be what you are viewing.